### DISCUSSION TITLE
236 Not using GPU

### DISCUSSION THREAD

**Discussion by andyh1213** [openvinotoolkit/openvino_notebooks#1096] [Q&A]
I have run notebook 236 without any changes to the code and it doesn't look like it is using my iGPU at all. The line:

output_file = generate_video(ov_pipe_inpaint, zoom_prompt.value, zoom_negative_prompt.value, 7.5, zoom_num_steps.value, zoom_num_frames.value, mask_width.value, zoom_seed.value, zoom_in.value)

uses substantial CPU when running but my GPU shows 0%.

Notebook 225 did appear to use my GPU during rendering.

I'm on a laptop with a i5-9300H CPU

**Comment by raymondlo84** [comment]
Yes, it is pointed to CPU by default for 236. You can change the "CPU" flag to "GPU" in the code.

**Reply by andyh1213** [reply]
Thanks, I was able to get that working for the text_enc, vae_decoder, and vae_encoder, but on the unet I am getting this:

RuntimeError: Check 'layout.bytes_count() <= get_device_info().max_alloc_mem_size' failed at C:\Jenkins\workspace\private-ci\ie\build-windows-vs2019\b\repos\openvino\src\plugins\intel_gpu\src\runtime\ocl\ocl_engine.cpp:135:
[GPU] Exceeded max size of memory object allocation: Requested 1698693120 bytes but max alloc size is 1660907520 bytes

I assume that since I'm on an integrated GPU (Intel HD 630) that this model is just too large to run there?

With the text encoder and vae set to GPU and unet to CPU it worked fine but I didn't really see any speed increase over just running them all on CPU.
